src/sensors/elec.rs

```diff
@@ -0,0 +1,28 @@
+/// Simulate sensor electronics using charge and time smearing
+
+// use petalo::io::mcreaders::MCSensorHit;
+use rand::random;
+
+
+pub fn filter_pde(pde: f32) -> bool {
+    random::<f32>() < pde
+}
```

Comment:
I think that getting something that works, however hacky, as a stepping stone towards something that is more solid and correct, is absolutely the right way to go.

Reply:
I'm going through all the unresolved conversations, checking what has been done and what hasn't, and marking them resolved if they have been addressed. In some cases, because of changed names of functions and files, it takes quite some effort to track down the relevant code. It would be helpful if you would mark things as resolved as you resolve them.